tests/react-a11y-media-captions/test.tsx.lint

```diff
@@ -0,0 +1,61 @@
+
+const SomeComponent = () =>
+    <video width="320" height="240">
+        <source src="sitting_and_smiling_234.mp4" type="video/mp4"/>
+        <track src="subtitles_en.vtt" kind="subtitles" srclang="en" label="English"/>
+        <track src="sitting_and_smiling_234.vtt" kind="captions" srclang="en" label="English"/>
+        <track src="myvideo_en.vtt" kind="description" srclang="en" label="English"/>
+    </video>;
+
+const SomeComponent = () =>
+<video width="320" height="240">
+    <source src="sitting_and_smiling_232.mp4" type="video/mp4"/>
+    <track src="subtitles_en.vtt" kind={subtitles} srclang="en" label="English"/>
```

Comment:
Hmm... Looks like I've missed what will be checked inside `JsxExpression`.
I would expect simple check for string `kind={'subtitles'}` but not for variable name.
Without type info (I don't think it is needed for initial implementation) it would be great to check only strings inside `JsxExpression`.

Also would be nice to skip errors if there are spread attributes on `track` elements (see [reasoning here](https://github.com/Microsoft/tslint-microsoft-contrib/pull/849#discussion_r272595545))

Reply:
Hi @IllusionMH 
I'm sorry for being away for so long...  🙏 

changed expression check according to your comments.
also, added `Spread Attribute` detection on track elements, so the following test should pass:

```const SomeComponent = () =>
    <video width="320" height="240">
        <source src="sitting_and_smiling_260.mp4" type="video/mp4"/>
        <track {...props} />
        <track src="subtitles_en.vtt" kind="subtitles" srclang="en" label="English"/>
    </video>;```